R/TealReportCard.R

```diff
@@ -40,14 +41,18 @@ TealReportCard <- R6::R6Class( # nolint: object_name_linter.
       attr_fs <- attr(fs, "formatted")
       checkmate::assert_character(attr_fs, null.ok = TRUE)
 
-      if (is.null(attr_fs)) {
+      if (length(fs) != 0) {
```

Comment:
worth adding about if length 0 then nothing is added to the card?

Reply:
updated